src/server.h

```diff
@@ -1863,12 +1898,14 @@ struct redisServer {
     int repl_timeout;               /* Timeout after N seconds of master idle */
     client *master;     /* Client that is master for this slave */
     client *cached_master; /* Cached master to be reused for PSYNC. */
+    client *psync_master; /* Client used for psync during rdb load */
     int repl_syncio_timeout; /* Timeout for synchronous I/O calls */
     int repl_state;          /* Replication status if the instance is a slave */
     off_t repl_transfer_size; /* Size of RDB to read from master during sync. */
     off_t repl_transfer_read; /* Amount of RDB read from master during sync. */
     off_t repl_transfer_last_fsync_off; /* Offset when we fsync-ed last time. */
     connection *repl_transfer_s;     /* Slave -> Master SYNC connection */
+    connection *repl_full_sync_s;    /* Master FULL SYNC connection (RDB download) */
```

Comment:
maybe this term is better?
```suggestion
    connection *repl_2nd_chan_s;    /* Master FULL SYNC connection (RDB download) */
```

Reply:
changing to `repl_rdb_transfer_s`